cmd/juno/juno.go

```diff
@@ -2,13 +2,27 @@ package main
 
 import (
 	"fmt"
+	"runtime/debug"
 
 	"github.com/NethermindEth/juno/node"
 	"github.com/NethermindEth/juno/utils"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+var version = "v0.1.0"
+
+var Commit = func() string {
+	if info, ok := debug.ReadBuildInfo(); ok {
+		for _, setting := range info.Settings {
+			if setting.Key == "vcs.revision" {
+				return setting.Value
+			}
+		}
+	}
+	return ""
+}()
+
```

Comment:
Is there a way to get a tag instead of a commit?

Reply:
we can use `git describe --tags`

Reply:
done

Reply:
```
$ ./build/juno --version
juno version legacy-260-ge65d841
```

Reply:
Do you know what the 260 means, is it the commit difference from the main?

Reply:
Yeap, number of new commits after `legacy`